plugins/deployment_manager/DeploymentManager.ts

```diff
@@ -286,8 +286,6 @@ export class DeploymentManager {
       debug(`Retrying with retries left: ${retries}, wait: ${wait}`);
       debug('Error is: ', e);
       if (retries === 0) throw e;
-      // XXX to be extra safe, we can also get the signer transaction count and figure out the next nonce
-      this._signers = [];
```

Comment:
hmm this is needed for our deploy scripts to be able to retry hanging txns successfully. I guess I should have made the intention more explicit, but we clear out signers to force `getSigners()` to rebuild a new Signer with a new provider.